packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -240,6 +240,31 @@ export default declare((api, opts) => {
     }
   }
 
+  function hasMoreThanOneBinding(node) {
+    // ArrayPattern
+    if (node?.elements) {
```

Comment:
After filtering out the `null` in array patterns, I actually think `node` is always non-nullish.

Reply:
Sure - I'm happy to remove it but not sure if it's worth being defensive here so we don't break builds.